packages/admin-client/src/types/index.ts

```diff
@@ -0,0 +1 @@
+export type AppEnv = "Local Dev" | "Test" // TODO: Add more environments
```

Comment:
@brankoEISI what other env variables we have for `REACT_APP_ENVIRONMENT`?This needs to be updated and bath path function updated based on this as well.